_integration/testsuite/httpproxy/011-retry-policy-validation.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright 2020 VMware, Inc.
```

Comment:
we've recently switched all existing copyright headers to `// Copyright Project Contour Authors`, please update this one.